src/Marketplace/docs/Move-AzMarketplacePrivateStoreCollectionOffer.md

```diff
@@ -0,0 +1,251 @@
+---
+external help file:
+Module Name: Az.Marketplace
+online version: https://docs.microsoft.com/powershell/module/az.marketplace/move-azmarketplaceprivatestorecollectionoffer
+schema: 2.0.0
+---
+
+# Move-AzMarketplacePrivateStoreCollectionOffer
+
+## SYNOPSIS
+transferring offers (copy or move) from source collection to target collection(s)
```

Comment:
What's the difference with `Copy-AzMarketplacePrivateStoreCollectionOffer`? Is the operation always be `Move`?

Reply:
only copy, fixed